### ISSUE TITLE
Wee Little Typo

### ISSUE THREAD

**Issue by jakthom** [fishtown-analytics/docs.getdbt.com#989]
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

`Superfluous` is spelled `superflous` here: 

https://docs.getdbt.com/docs/building-a-dbt-project/metrics#declaring-a-metric

### What part(s) of the page would you like to see updated?

```
metrics:
  - name: new_customers
    label: New Customers
    model: ref('dim_customers')
    description: "The number of paid customers using the product"

    type: count
    sql: user_id # superflous here, but shown as an example
```

-> 

```
metrics:
  - name: new_customers
    label: New Customers
    model: ref('dim_customers')
    description: "The number of paid customers using the product"

    type: count
    sql: user_id # superfluous here, but shown as an example
```

### Additional information

<img width="687" alt="Screen Shot 2022-01-03 at 11 00 02 AM" src="https://user-images.githubusercontent.com/2599880/147952408-48868e40-f3f0-4509-849f-4ec55d421144.png">

**Comment by joellabes** [comment]
Thanks for spotting this @jakthom! Would you be interested in opening a PR to change this directly? At the bottom of every Docs page is a [edit this page](https://github.com/fishtown-analytics/docs.getdbt.com/edit/current/website/docs/docs/building-a-dbt-project/metrics.md) link where you can tweak it inline

**Comment by jakthom** [comment]
@joellabes totally! 

https://github.com/dbt-labs/docs.getdbt.com/pull/996

**Comment by runleonarun** [comment]
Closed with #966 Much appreciated!
